### PR TITLE
Update privacy_notice.md

### PR DESCRIPTION
**Pull request by InezMc**
Add device managment to Element Privacy Policy

### CODE REVIEW

**Review by deniseal — APPROVED** [review]
lgtm, approved

**Comment by Johennes** [comment]
The new device manager has started rolling out behind labs flags. Could we release this PP update?

**Comment by InezMc** [comment]
> The new device manager has started rolling out behind labs flags. Could we release this PP update?

Hi, we are just waiting for approval from @AmandineLP

**Review by AmandineLP — APPROVED** [review]

**Pull request merged**
